templates/common/bin/kolla_extend_start

```diff
@@ -12,6 +12,13 @@ EOF
 
 }
 
+# Temporary workaround until the glance-api image in tcib is fixed
```

Comment:
@konan-abhi we remove L15-L21 after [1] lands.

[1] https://github.com/openstack-k8s-operators/tcib/pull/262

Reply:
need to change now!!

Reply:
Done!